data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -78,6 +78,41 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f config-bootstrap.done ]
+then
+    record_service_stage_start "config-bootstrap"
+	echo "Rendering cluster config manifests..."
+
+	rm --recursive --force config-bootstrap
+
+	ADDITIONAL_FLAGS=()
+	if [ -f "$PWD/manifests/cloud-provider-config.yaml" ]; then
+		ADDITIONAL_FLAGS+=("--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml")
+	fi
+	{{- if .FeatureSet }}
+	ADDITIONAL_FLAGS+=("--feature-set={{.FeatureSet}}")
+	{{- end}}
+
+	bootkube_podman_run \
+		--name config-render \
+		--volume "$PWD:/assets:z" \
+		"${CONFIG_OPERATOR_IMAGE}" \
+		/usr/bin/cluster-config-operator render \
+		--cluster-infrastructure-input-file=/assets/manifests/cluster-infrastructure-02-config.yml \
+		--cloud-provider-config-output-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml \
+		--config-output-file=/assets/config-bootstrap/config \
+		--asset-input-dir=/assets/tls \
+		--asset-output-dir=/assets/config-bootstrap \
+		--featuregate-manifest=/assets/manifests/99_feature-gate.yaml \
```

Comment:
Does this file actually have to be called this? In other render commands I've seen they search through the manifests folder and look at a partial object meta to identify if the file is a featuregate. Pre the installer supporting feature gates, it could have been called whatever I liked, nothing to say that's not still the case for some who have set up clusters without FG support in the installer before

Reply:
This is a good point, spelling it out a bit more...

This PR will make the installer [always lay down a feature set manifest](https://github.com/openshift/installer/pull/6990/files#diff-0e32effcb5ed12fa66de8854ba39a725fd0a98675481abeae39ce71b25ccb721L47). If there is a conflicting feature set manifest the installer will throw an error pre-install. 

It would definitely be possible for a user to delete the installer-generated manifest. But in those cases only the feature set spec would be set and not the status, right?

Reply:
A quick scan of the release repo leads me to:
* https://github.com/openshift/release/blob/bb21ccca00ad923ea66912c5897335d48087851d/ci-operator/step-registry/ipi/conf/techpreview/ipi-conf-techpreview-commands.sh#L4
* https://github.com/openshift/release/blob/bb21ccca00ad923ea66912c5897335d48087851d/ci-operator/step-registry/ipi/conf/customnoupgrade/ipi-conf-customnoupgrade-commands.sh#L14
Both of which, IIUC, set up a feature gate manifest under a custom name during the openshift-install manifest stage. The current logic wouldn't pick these up and would invalidate all techpreview and customnoupgrade CI jobs we have.

Either we need to rename those to fit the pattern, or fix the logic to run a search in CCO of the manifest dir (the latter being the preferred long term IMO)